.github/workflows/linters.yml

```
@@ -98,3 +98,12 @@ jobs:
       - uses: mszostok/codeowners-validator@v0.4.0
         with:
           checks: "files,duppatterns"
+  lint-pr-url:
+    if: ${{ github.event.pull_request }}
```

```suggestion
    if: github.event.pull_request.draft == false
```
Isn't this supposed to be the check here (based on the above jobs)?

---

I meant to check only pull request (it wouldn't work on push, because it needs a PR URL), and I believe it's fine to have it running on draft PRs because it doesn't produce errors.